api/src/main/java/org/eclipse/microprofile/health/HealthCheckResponse.java

```diff
@@ -165,8 +182,13 @@ public Status getStatus() {
         return status;
     }
 
+    @Deprecated(since = "3.1")
```

Comment:
also in Javadoc mention deprecation and point to what will happen in next releases